### PR TITLE
DBR-321 - Fix issue with browser language

### PR DESCRIPTION
**Pull request by andresmoschini**
Hi team, 

I received a message from support area, about a user that cannot use our signup page. 

I am pretty sure that his browser has a different language configured than English or Spanish.

So, in my opinion, it should fix the issue.

Could you review the changes?

I promise to add a test for this issue today, but I want to merge it soon in order to deploy the fix to production ASAP.

### CODE REVIEW

File: src/i18n/DopplerIntlProvider.js [hunk 1]
```diff
@@ -14,7 +14,10 @@ const messages = {
 addLocaleData([...en, ...es]);
 
 export default ({ locale, children }) => (
-  <IntlProvider locale={locale || 'en'} messages={flattenMessages(messages[locale || 'en'])}>
+  <IntlProvider
+    locale={locale || 'en'}
+    messages={flattenMessages(messages[locale || 'en'] || messages['en'])}
```
Comment (fcoronelmakingsense) [inline src/i18n/DopplerIntlProvider.js]: only format change?

File: src/index.js [hunk 2]
```diff
@@ -21,7 +21,9 @@ if (document.querySelector('body').setActive) {
 }
 
 // TODO: this hardcoded data will depend by the app language
-const locale = navigator.language.toLowerCase().split(/[_-]+/)[0] || 'en';
+const localeFromNavigator = navigator.language.toLowerCase().split(/[_-]+/)[0] || 'en';
+// TODO: remove language validation code
+const locale = ['es', 'en'].includes(localeFromNavigator) ? localeFromNavigator : 'en';
```
Comment (fcoronelmakingsense) [inline src/index.js]: good catch. Maybe you right they have another language

File: src/i18n/DopplerIntlProvider.js (hunk 1 above)
Comment (andresmoschini) [reply src/i18n/DopplerIntlProvider.js]: No, now when `messages[locale || 'en']` is _falsy_ we will use `messages['en']`

**Comment by andresmoschini** [comment]
Thanks for your reviews! I really appreciate it.

**Pull request merged**